Template/azuredeploy.json

```diff
@@ -73,7 +80,8 @@
         "storageAccountId": "[concat(resourceGroup().id, '/providers/', 'Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]",
         "iotHubOwnerPolicyName": "iothubowner",
         "appInsightName": "[concat(parameters('uniqueSolutionPrefix'), 'insight')]",
-        "redisCacheName": "[concat(parameters('uniqueSolutionPrefix'), 'redis')]"
+        "redisCacheName": "[concat(parameters('uniqueSolutionPrefix'), 'redis')]",
+        "logAnalyticsName": "[concat('log-', parameters('uniqueSolutionPrefix'))]"
```

Comment:
the convention so far had been to add a suffix not a prefix. Is there a specific reason to change?

Reply:
I was trying to adhere to the [recommendations](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations), but I agree, that doesn't make any sense here to change the convention. Will use a suffix instead.

Reply:
c109df03f4c308f99f2d007bca0df97a923eacbe